pppd/Makefile.am

```diff
@@ -181,3 +192,6 @@ pppd_LDADD = $(pppd_LIBS)
 
 EXTRA_DIST = \
     ppp.pam
+
+TESTS = $(check_PROGRAMS)
```

Comment:
I'm out of date with the syntax of these files, but if I follow it right then this will execute `check_PROGRAMS` (a-la Bash) and assign the output to `TESTS`? What would happen if `check_PROGRAMS` is empty?

Reply:
I just ran this and commented out the $(check_PROGRAMS) leaving TESTS empty, and sure enough it didn't run any tests as expected. 

Once can configure without MSCHAP and without PEAP using configure, and in that case; none of the unit-tests would run. More likely is that you disable PEAP and also disable the unit-test.